src/UserTest.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.Test;
+import org.junit.function.ThrowingRunnable;
+
+import static org.junit.Assert.*;
+
+public class UserTest {
+    private String login;
+    private String email;
+
+    @Test
+    public void testUser1() {
+        User user = new User("Petr", "petr@email.ru");
+        assertNotNull(user.getLogin());
+        assertNotNull(user.getEmail());
+    }
+
+    @Test
+    public void testUser2() {
+        User user = new User();
+        assertNull(user.getLogin());
+        assertNull(user.getEmail());
+    }
+
+    @Test
+    public void testUser3() {
+        // User user = new User("Petr", "petr@email.ru");
+        assertThrows(RuntimeException.class, () -> new User("Petr", "petremail.ru"));
+        // assertNotEquals("", user.getEmail());
+    }
+
+    @Test
+    public void testUser4() {
+        User user = new User("Petr", "petr@email.ru");
+        assertNotEquals(user.getLogin(), user.getEmail());
```

Comment:
Тест должен проверять, что будет выброшено исключение, если передать эквивалентные логи и почту. Примерно вот так:
```suggestion
        assertThrows(IllegalStateException.class, () -> new User("Petr", "Petr"));
```